src/components/NewWorkspaceModal.js

```diff
@@ -153,7 +171,7 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
   const destLocationType = bucketLocation === defaultLocation ? locationTypes.default : locationTypes.region
 
   return Utils.cond(
-    [loading, () => spinnerOverlay],
+    [loading || billingProjects === undefined, () => spinnerOverlay],
```

Comment:
This addresses a bug in production. Currently, when you select the "Clone" operation, you briefly see the dialog below pop up before the "Clone Workspace" dialog. This is because the code thinks that there are no billing projects configured until the billing projects are fetched, and it is possible for the "loading" flag to get set to false before the billingProjects state variable has been set (the order of operations does not seem consistent).

Dialog we don't want to see flicker up:
![image](https://user-images.githubusercontent.com/484484/167433601-21a11dac-b293-4e5d-b811-b8134a61b0b9.png)